tests/functional/infra_lv_create/site.yml

```diff
@@ -0,0 +1 @@
+../../../site.yml.sample
```

Comment:
not sure why you had to create this symlink since this is never called in `[testenv:infra_lv_create]`

Reply:
You're right, it`s useless.
It appeared because --noprovisioning flag didn`t work and it was resolved by using vagrant==1.8.6.

Reply:
I deleted the symlink, do I need to create a new pull request without redundant commit?